music21/analysis/floatingKey.py

```diff
@@ -70,38 +86,55 @@ def __init__(self, s=None):
 
         self.weightAlgorithm = divide
         if s.hasPartLikeStreams():
-            p = s.parts.first()
+            p = s.iter.parts.first()
```

Comment:
This is just adding safety in case `s` is a generic `stream.Stream` (only Score has this property)